src/components/Tabs/Tabs.mdx

```diff
@@ -75,26 +76,34 @@ Resources:
 
 _Note: this is the recommended approach._
 
-We recommended passing in the data for the `Tabs` component through the `data`
-prop. This allows the consumer to pass in well-formed data and always generate
-the correct DOM.
-
-Internally in the Reservoir Design System (DS) `Tabs` component:
-
-- we map through the data array and generate the appropriate components needed
-  for the layout.
-- adds the carousel feature for the mobile view.
-
-Note that HTML is allowed in the `content` property in each object as part of
-the string or as straight HTML.
+- It is recommended touse the `tabsData` prop to populate the component. This allows
+  the consumer to pass in well-formed data and always generate the correct DOM.
+- The `Tabs` component will map through the `tabsData` array and generate the
+  appropriate components needed for the layout.
+- The `Tabs` component will add the carousel feature for the mobile view.
```

Comment:
Nit: it doesn't seem like we consistently use bullet points - do we need them here or elsewhere in this page?

Reply:
I updated the page to simplify and use more bullets.